vispy/visuals/__init__.py

```diff
@@ -22,8 +22,8 @@
 from .isoline import IsolineVisual  # noqa
 from .isosurface import IsosurfaceVisual  # noqa
 from .line import LineVisual  # noqa
-from .line_plot import LinePlotVisual  # noqa
-from .markers import MarkersVisual, marker_types  # noqa
+# from .line_plot import LinePlotVisual  # noqa
+# from .markers import MarkersVisual, marker_types  # noqa
```

Comment:
You should be able to just merge from my scenegraph-update branch; no need to rebase/replay anything.